core/tls.cabal

```diff
@@ -160,6 +160,7 @@ Benchmark bench-tls
                    , QuickCheck >= 2
                    , tasty-quickcheck
                    , tls
+  ghc-options:       -Wall -fno-warn-unused-imports -fno-warn-orphans
```

Comment:
this only apply to bench-tls though. when writing tests/benchs, it's not a problem to have orphans (and sometimes it's the only way)